clients/config_client/config_client.go

```diff
@@ -228,6 +228,7 @@ func (client *ConfigClient) getConfigInner(param *vo.ConfigParam) (content strin
 	content = cache.GetFailover(cacheKey, client.configCacheDir)
 	if len(content) > 0 {
 		logger.Warnf("%s %s %s is using failover content!", clientConfig.NamespaceId, param.Group, param.DataId)
+		param.Content = content
```

Comment:
Why do  need to set param

Reply:
https://github.com/nacos-group/nacos-sdk-go/pull/668，there is a problem with the changes in this, and look at it after repair

Reply:
Fixed: https://github.com/nacos-group/nacos-sdk-go/pull/697